profile_bluesky/startup/instrument/devices/ad_rigaku_detector.py

```diff
@@ -1,9 +1,7 @@
 """
 ADRigaku UFXC area detector (EPICS)
-
 * detector name: -tba-
 * detector number: -tba-
-
```

Comment:
This blank line should stay also (it is part of the restructured text formatting)